exercises/series/series_test.exs

```diff
@@ -0,0 +1,57 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("series.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule StringSeriesTest do
+  use ExUnit.Case
+
+  test "slices of 1" do
+    assert StringSeries.slices("01234", 1) == ["0", "1", "2", "3", "4"]
+    assert StringSeries.slices("92834", 1) == ["9", "2", "8", "3", "4"]
+  end
+
+  @tag :pending
+  test "slices of 2" do
+    assert StringSeries.slices("01234", 2) == ["01", "12", "23", "34"]
+    assert StringSeries.slices("98273463", 2) == ["98", "82", "27", "73", "34", "46", "63"]
+    assert StringSeries.slices("37103", 2) == ["37", "71", "10", "03"]
+  end
```

Comment:
Not really. They were 3 separate tests I copied from the Ruby version and condensed into one.